crates/client/src/storage/fs.rs

```diff
@@ -70,10 +71,10 @@ impl FileSystemRegistryStorage {
     }
 
     fn package_path(&self, id: &PackageId) -> PathBuf {
-        self.base_dir.join(
+        self.base_dir.join(PACKAGE_LOGS_DIR).join(
             LogId::package_log::<Sha256>(id)
                 .to_string()
-                .replace(':', "/"),
+                .replace(':', "-"),
```

Comment:
As `WalkDir` is recursive, can we keep the algo directory?

Reply:
Sure.